lib/platform/gitlab/index.md

```diff
@@ -13,3 +13,8 @@ By setting the server version yourself, you save a API call that fetches the ser
 - Use `Draft:` MR prefix instead of `WIP:` prefix since `v13.2.0`
 - Do not truncate Markdown body to 25K chars since `v13.4.0`
 - Allow configure reviewers since `v13.9.0`
+
+## Multiple merge request assignees
+
+Due to licensing restrictions [multiple assignees](https://docs.gitlab.com/ee/user/project/issues/multiple_assignees_for_issues.html) are only available in GitLab Premium self-managed, GitLab Premium SaaS, and higher tiers.
```

Comment:
Do we want to mention that multiple assignees only works on some GitLab tiers in the `assignees` config option docs?

https://docs.renovatebot.com/configuration-options/#assignees

Reply:
I think this is enough, as gitlab users should know this in general.